### DISCUSSION TITLE
What about environment?

### DISCUSSION THREAD

**Discussion by Docbroke** [leo-arch/clifm#71] [General]
How clifm treats environment? What I have noticed is $PATH is respected but aliases  and functions in my .bashrc are unknown commands to clifm.

**Comment by leo-arch** [comment]
Hi @Docbroke. There are only a few values CliFM takes from the environment: PATH, CDPATH, HOME, DISPLAY, MANPAGER, INPUTRC, PWD, and TERM. Shell specific values are ignored for portability reasons. However, bash-like aliases can be imported using the `alias import` command.

**Comment by Docbroke** [comment]
Thanks. alias import is very useful, I was able to import almost all aliases I have in single go.
